tests/MongoDB.Bson.TestHelpers/JsonDrivenTests/JsonDrivenTest.cs

```diff
@@ -66,6 +66,11 @@ public virtual void Arrange(BsonDocument document)
                 SetArguments(document["arguments"].AsBsonDocument);
             }
 
+            if (document.Contains("error"))
+            {
+                _expectedException = new BsonDocument(); // any exception will do
```

Comment:
_expectedException is a document that contains the expectations of what the exception looks like, things like error codes and names.

An empty BsonDocument means there is nothing about the exception (other than that it was thrown) that needs to be asserted.